schema/region.json

```diff
@@ -0,0 +1,7 @@
+{
+    "Region": {
+        "description": "Identifier for a geographic region",
+        "type": "string",
+        "enum": ["CO", "AK", "PR", "HI"]
```

Comment:
I assume CO here is CONUS? Is there a use case for this enum where it could be Colorado?

Reply:
Yep, CO is for CONUS. Don't expect we will have products that are available by state, but if we do we'll need to make some changes. I think the most likely next step, is products that are custom local domains. Like a super high resolution wind model in the area surrounding a fire. Don't have a solution for that, but we'll deal with it when we need to.